apps/bfd-pipeline/bfd-pipeline-shared-utils/src/main/java/gov/cms/bfd/pipeline/sharedutils/PipelineTestUtils.java

```diff
@@ -212,6 +203,25 @@ public void truncateTablesInDataSource() {
     }
   }
 
+  /**
+   * Schemas that use mixed case use quotes and have their original case preserved but those without
+   * quotes are converted to upper case to be compatible with Hibernate/HSQLDB.
+   *
+   * <p>Note: This may need to be quoted on PostgreSQL. If so, since HSQL DB blows up if we quote
+   * them, this code may have to first check the DB platform. TBD.
```

Comment:
can we determine this before going forward by running against postgres locally?

Reply:
Thanks.  I tested with postgres and found problems.  These tests run correctly in HSQLDB and Postgresql now.